en/using-data-in-your-app.md

```diff
@@ -38,6 +38,83 @@ Check out the [full article on how to add a Swagger definition](open-api-swagger
 <img class="responsive-img" style="box-shadow: 5px -4px 13px 1px grey" src="./images/swagger-demo-original.gif" />
 <p style="text-align:center;">Swagger demo</p>
 
+## Using data source from local network sources
+You can now make requests to a localhost or private networks. This includes regular REST endpoints and Swagger endpoints. 
+
+<img class="responsive-img" style="box-shadow: 5px -4px 13px 1px grey" src="./images/internal-network-data-source.gif" />
+<p style="text-align:center;">Local network sources usage</p>
+
+### Troubleshooting
+### Unknown Error - Something went wrong when connecting
+Due to the essence of working with local networks this type of error requires additional work in order to identify if the local/private service fails due to CORS problems (Cross-Origin Requests) or something else.
+
+If `Unknown error` dialog appears while you add a local service, open the `dev tools` of your browser (Hit `F12`) and check the console/network tab for errors. The most probable reason would be CORS restrictions.
+
+<img class="responsive-img" style="box-shadow: 5px -4px 13px 1px grey" src="./images/internal-network-error.png" />
```

Comment:
I think it would be nice if this image also contains the open dev tools info, so the user can understand how to arrive to the same conclusions.